### DISCUSSION TITLE
Not working on OOS 13

### DISCUSSION THREAD

**Discussion by JReming85** [Mygod/VPNHotspot#496] [Q&A]
Some how, some way tmo is still detecting hotspot usage on a OnePlus 10 Pro, running android 13.

I have setup several VPNs,
1 - Google One VPN
2 - Several Openvpns
3 - Speed test IPVanish VPN

Was watching usage from the OpenVPN client and it looks like hotspot was bypassing it

**Comment by Mygod** [comment]
How did you determine hotspot was detected?

**Reply by JReming85** [reply]
Alert from my carrier that my allotted 5gb of highspeed was almost all used up.

I can track hotspot usage on their website

**Reply by Mygod** [reply]
Did you enable `wlan0` or whatever in the app after you enable hotspot? Can you verify that your WAN IP has been changed on connected clients?

**Reply by JReming85** [reply]
Looks like it is rotting through the VPN as the devices on the hotspot are getting the VPNs IP.

Will play around with it more later today to double check, but as of now looks like it may just be some sort of aggressive hotspot detection with TMobile.

I understand this apps main purpose is not to circumvent hotspot usage limits but any help would be much appreciated.

**Comment by JReming85** [comment]
I tried with both the temp and normal hotspot setting, with and without
wlan2 on for the normal hotspot.

I also set it to use tun0 with no fallback.

Can't confirm IP, will check in the morning/tomorrow as it's pretty late
here and I don't get any service at my hotel.

**Reply by Mygod** [reply]
Sounds good. Make sure you've also disabled IPv6 tethering for other leakage.
